### ISSUE TITLE
Results Stage1 KIT Dataset

### ISSUE THREAD

**Issue by AlessioSam** [ChenFengYe/motion-latent-diffusion#50]
Hi,
thank you for your work, I find it brilliant and really useful for future developments in this field!

Regarding the HumanML3D dataset, the results are easily replicable. 
However when trying to train the model on the KIT dataset I am having trouble getting your results, similar to issue #44.

- I changed as you rightly said in a previous issue (#13) some lines in the config ('humanml' -> 'kit'), leaving the field JOINT_TYPE: 'humanml3d' unchanged.
- I have repeatedly checked that I have downloaded the correct dataset and dependencies.

In stage1, what happens is that the model seems not to learn to reconstruct joints and features. 

Is it possible to have your checkpoint so I can check that indeed the dataset is not corrupted? Or if you have any other advice (hyperparameters, ...)

After 20k epochs I have:
FID: 26.7
R_precision_top_1: 0.199
AVE_mean_pose: 2145
AVE_mean_joints: 79551

In contrast, the metrics for GT are correct.

Looking forward to your reply! 
Thanks,
Alessio 
sampieri@diag.uniroma1.it

**Comment by weleen** [comment]
@AlessioSam I have the same problem, have you solved it?

**Comment by ChenFengYe** [comment]
@weleen @AlessioSam Hi Weleen and AlessioSam, could you provide more details about this implementation?
The FID metric shows that this training is broken.
Is there anything about loss?

**Comment by weleen** [comment]
@ChenFengYe 
<img width="596" alt="image" src="https://github.com/ChenFengYe/motion-latent-diffusion/assets/7327734/07a04990-b295-4542-9a3e-1bb61cd80ce3">
This is my test results.
